src/tables.rs

```diff
@@ -42,7 +42,7 @@ pub enum GeneralCategory {
 }
 
 #[allow(dead_code)]
-pub const GENERAL_CATEGORY: &'static [(u32, u32, GeneralCategory)] = &[
+pub const GENERAL_CATEGORY: &[(u32, u32, GeneralCategory)] = &[
```

Comment:
This is generated code (see the comment at the top), so best not to modify it by hand.